dvc/command/checkout.py

```diff
@@ -56,5 +57,6 @@ def add_parser(subparsers, parent_parser):
         nargs="*",
         help="DVC-files to checkout. Optional. "
         "(Finds all DVC-files in the workspace by default.)",
+        type=lambda s: decode(s, "utf8"),
```

Comment:
I'm pretty sure this is caused by the `sys.argv` returning things encoded. So putting something like
```
argv = [decode(arg, "utf-8") for arg in sys.argv[1:]]
args = parser.parse_args(argv)
```
in dvc/cli.py would do the same thing without needing to modify each cmd.

Reply:
nice trick, I'll try it